feat(guide): Collection of Custom Formats - HDR Formats - Improve description (#1760)
TRaSH-Guides/Guides: includes/cf-descriptions/dv-hdr10plus.md

*This Custom Format will only match on* `DV HDR10+`

> This provides Dolby Vision with HDR10+ fallback, while HDR10+ also has HDR10 fallback..

Reviewer: ```suggestion
This format provides Dolby Vision *with* HDR10+ fallback (with HDR10+ having native HDR10 fallback).
```
